kext/FileNVRAM/FileNVRAM.cpp

```diff
@@ -47,15 +49,38 @@ void FileNVRAM::setPath(OSString* path)
 
 bool FileNVRAM::start(IOService *provider)
 {
+    char peBuf[256];
+    mReadOnly      = false;
     bool earlyInit = false;
     
-    LOG(NOTICE, "start() called (%d)\n", mInitComplete);
+    if(PE_parse_boot_argn("-NoFileNVRAM", PEbuf, sizeof peBuf))
```

Comment:
I'd suggest moving the "-NoFileNVRAM" and "-FileNVRAMro" strings to the header file so all of the options are in one spot.